src/classes/ADDR_Contact_TDTM.cls

```diff
@@ -326,25 +327,26 @@ public class ADDR_Contact_TDTM extends TDTM_Runnable {
 	            
 	            //Copy address info to parent Account
 	            ADDR_Addresses_UTIL.copyAddressStdSObj(originalContact, 'Mailing', acc, 'Billing');
-	            objectsToUpdate.add(acc);
+	            accountsToUpdate.add(acc);
 	            
 	            //Copy address info to each sibling - we need to exclude the one the trigger is running on!
 	            for(Contact contact : acc.Contacts) {
 	                if(contact.ID != originalContact.ID && !contact.is_Address_Override__c) {
 	                   contact.Primary_Address_Type__c = originalContact.Primary_Address_Type__c;
 	                   ADDR_Addresses_UTIL.copyAddressStdSObj(originalContact, 'Mailing', contact, 'Mailing');
-	                   objectsToUpdate.add(contact);
+	                   contactsToUpdate.add(contact);
 	                }
 	            }
 	        //If the address info is the same, just update the Current_Address__c field in the original contact
             } else if(originalContact != null && ADDR_Addresses_UTIL.isContactAccAddrInfoSame(originalContact, acc)) {
                 originalContact.Current_Address__c = acc.Current_Address__c;
-                objectsToUpdate.add(originalContact);
+                contactsToUpdate.add(originalContact);
             }
         }
         //explicitly doing the update, because if we put the records in DmlWrapper we get a "duplicate id in list" error due to
         //the method updateAccountPrimaryContact in ACCT_IndividualAccounts_TDTM also putting them there.
-        update objectsToUpdate;   
+        update accountsToUpdate;
+        update contactsToUpdate;
```

Comment:
The PR makes a lot of sense, @kyleschmid. Good catch. And thank you for your contribution. 

Now that we have accounts and contacts in two separate lists, would you mind combining them into one for the update? The order should be preserved, and switching from one object type to another should occur only once. Thanks!